package.json

```diff
@@ -0,0 +1,61 @@
+{
+	"name": "cli",
+	"version": "0.0.0",
+	"license": "MIT",
+	"bin": {
+		"asyncapi": "dist/cli.js",
+		"@asyncapi/cli": "dist/cli.js"
```

Comment:
I think that to be able to run it as a cli, yes, but I will test it and take a look at it

Reply:
@jotamusik I think `asyncapi` is needed only

look at https://github.com/asyncapi/generator/blob/master/package.json#L6-L9

nobody will use CLI with `@asyncapi/cli` name

Reply:
Maybe `a2s` as alias?

```suggestion
		"@asyncapi/cli": "dist/cli.js",
		"a2s": "dist/cli.js"
```

Reply:
I would avoid any other aliases really, `asyncapi` is the best one and for the rest you have bash aliases really. I'd like to avoid https://github.com/asyncapi/generator/issues/467

Reply:
done!